base/ltfssaxes.dtx

```diff
@@ -1195,59 +1208,110 @@
+\DeclareFontShapeChangeRule {n}{up}  {n}   {}
+%    \end{macrocode}
+%    For the \texttt{ssc} shape we make the following general
+%    assumptions: if \texttt{ssc}\meta{X} exists then
+%    \texttt{sc}\meta{X} and \meta{X} also exist in the font.  If the
+%    \texttt{ssc}\meta{X} shape doesn't exist but the user had asked
+%    for \texttt{ssc} we try to replace it by \texttt{sc} because we
+%    then assume that the current font family simply doesn't have any
+%    \texttt{ssc} shapes.  However, if we are already in some
+%    \texttt{ssc} shape and a change is requested we know that at
+%    least some \texttt{ssc} shapes exist for the current font family,
+%    so rather than falling back to some \texttt{sc} we try to stay
+%    within \texttt{ssc} changes in a fallback situation.
+% \changes{v1.0k}{2024/12/11}{Add \texttt{ssc} shape change
+%    rules (gh/1581)}
+%    \begin{macrocode}
+\DeclareFontShapeChangeRule {n}{ssc} {ssc} {sc}
 %    \end{macrocode}
 %
 %    \begin{macrocode}
 %\DeclareFontShapeChangeRule {it}{n}   {n}     {}
 %\DeclareFontShapeChangeRule {it}{it}  {it}    {}
 \DeclareFontShapeChangeRule {it}{sl}  {sl}    {it}
-%\DeclareFontShapeChangeRule {it}{sw}  {sw}    {}
+\DeclareFontShapeChangeRule {it}{sw}  {sw}    {it}
 %    \end{macrocode}
 %    If neither \texttt{scit} nor \texttt{scsl} exist then \texttt{sc}
 %    will be used as a fallback albeit with a log entry, so except for
 %    the latter there will be no change for CM or Latin Modern fonts.
 %    \begin{macrocode}
 \DeclareFontShapeChangeRule {it}{sc}  {scit}  {scsl}
 \DeclareFontShapeChangeRule {it}{ulc} {it}    {}
-\DeclareFontShapeChangeRule {it}{up} {n}      {}
+\DeclareFontShapeChangeRule {it}{up}  {n}     {}
+\DeclareFontShapeChangeRule {it}{ssc} {sscit} {scit} 
 %    \end{macrocode}
 %
 %    \begin{macrocode}
 %\DeclareFontShapeChangeRule {sl}{n}   {n}     {}
 \DeclareFontShapeChangeRule {sl}{it}  {it}    {sl}
 %\DeclareFontShapeChangeRule {sl}{sl}  {sl}    {}
-%\DeclareFontShapeChangeRule {sl}{sw}  {sw}    {}
+\DeclareFontShapeChangeRule {sl}{sw}  {sw}    {it}
 \DeclareFontShapeChangeRule {sl}{sc}  {scsl}  {scit}
 \DeclareFontShapeChangeRule {sl}{ulc} {sl}    {}
-\DeclareFontShapeChangeRule {sl}{up} {n}      {}
+\DeclareFontShapeChangeRule {sl}{up}  {n}      {}
+\DeclareFontShapeChangeRule {sl}{ssc} {sscsl}  {scsl}
 %    \end{macrocode}
 %
 %    \begin{macrocode}
 %\DeclareFontShapeChangeRule {sc}{n}   {n}     {}
 \DeclareFontShapeChangeRule {sc}{it}  {scit}  {scsl}
 \DeclareFontShapeChangeRule {sc}{sl}  {scsl}  {scit}
-\DeclareFontShapeChangeRule {sc}{sw}  {scsw}  {sw}
+\DeclareFontShapeChangeRule {sc}{sw}  {scsw}  {scit}
 %\DeclareFontShapeChangeRule {sc}{sc}  {sc}    {}
-\DeclareFontShapeChangeRule {sc}{ulc} {n}     {}
+\DeclareFontShapeChangeRule {sc}{ulc}  {n}    {}
 %    \end{macrocode}
 %
 %    The next rule might be a bit surprising and rightly so. Correct
 %    would be that \texttt{sc} is not affected by \texttt{up}, i.e.,
```

Comment:
Maybe:
It would be more correct if \text{sc}
were not . . . 

(or "was" if you don't like the subjunctive:-)